src/documents/IoCTestingFramework.md

```diff
@@ -122,151 +116,422 @@ fluid.tests.globalCatTest = function (catt) {
 };
 ```
 
-### `modules`, `tests` and `sequence` ###
-
-The standard structure inside a `fluid.test.testCaseHolder` shows an outer layer of containment, `modules`, corresponding to a
-QUnit `module`, and then a entry named `tests`, holding an array of structures corresponding to a QUnit `testCase`. Here we define a single
-test case which holds a single *fixture record* which executes a global function, `fluid.tests.globalCatTest` which makes one jqUnit assertion.
-This is not realistic for a normal use of the framework, which as we mentioned in the introduction only achieves its value when testing a sequence
-of fixtures back to back. Normally, the entry within `tests` will instead hold an entry named `sequence` which holds an array of fixture records
-representing sequence points to be attained by the test case.
-An example of the `sequence` entry appears below in the [asyncTester example](#testcaseholder-demonstrating-sequence-record).
-
-In order to run this test case, we can either simply construct an instance of the environment tree by calling `fluid.tests.myTestTree()`,
-or submit its name to the global driver function `fluid.test.runTests` as `fluid.test.runTests("fluid.tests.myTestTree")`.
-The latter method should be used when running multiple environments within the same file to ensure that their execution is properly serialised.
+<div class="infusion-docs-note"><strong>Note:</strong> In straightforward cases, the test environment component
+(e.g. `fluid.tests.myTestTree`), and the test fixture component (e.g. `fluid.tests.catTester`) can be written
```

Comment:
Because this is within a block of HTML, the code references should be written inside `<code>` tags.